cves/CVE-2013-0868.yml

```diff
@@ -1,368 +1,406 @@
+
+  For readability, we hard-wrap multi-line strings at 80 characters. This is
+  not absolutely required, but appreciated.
+  === End YAML Primer===
+curated_instructions: |
+  If you are manually editing this file, then you are "curating" it.
+  This will enable additional integrity checks on this file to make sure you
+  fill everything out properly.
+
+  IMPORTANT: If you are a student, we cannot accept your work as finished unless
+  curation_level is properly set.
+
+  The latest curation level is 1.0.
+  If you are curating this vulnerability, set it to the latest curation level.
+curation_level: 1.0
+reported_instructions: |
+  What date was the vulnerability reported to the security team? Look at the
+  security bulletins, bug reports, commit messages of the fix.
+
+  Reported date is likely the same as announced and published. Leave blank if
+  you can't find it out.
+
+  Please enter your date in YYYY-MM-DD format.
+reported_date: 2013-11-23
+announced_instructions: |
+  Was there a date that this vulnerability was announced to the world? You can
+  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
+
+  A good source of this for FFmpeg are their version numbers traced to tags:
+
+    https://ffmpeg.org/security.html
+    https://github.com/FFmpeg/FFmpeg/releases
+
+
+  Please enter your date in YYYY-MM-DD format.
+announced_date: 2013-02-05
+published_instructions: |
+  Is there a published fix or patch date for this vulnerability?
+  Please enter your date in YYYY-MM-DD format.
+published_date: 2013-02-05
+description_instructions: |
+  You can get an initial description from the CVE entry on cve.mitre.org. These
+  descriptions are a fine start, but they can be kind of jargony.
+
+  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
+  read to anyone with some programming experience. We can always pull up the NVD
+  description later to get more technical.
+
+  Try to still be specific in your description, but remove project-specific
+  stuff. Remove references to versions, specific filenames, and other jargon
+  that outsiders to this project would not understand. Technology like "regular
+  expressions" is fine, and security phrases like "invalid write" are fine to
+  keep too.
+
+  Your target audience is people just like you before you took any course in
+  security
+description: |
+  When parsing audio visual data for Huffyuv codec files, various conditions such
+  as invalid input format, empty lengths, and other can occur. If one of these are hit,
+  an error code is returned, indicating something went wrong.
+  In this case the return codes from functions in the Huffyuv decoder are never checked.
+  Additionally, some basic checks, such as ensuring the file isn't empty, are never made.
+  Together these allow an attacker to sneak an empty or invalid file in, which will result 
+  in an out of bounds write. This write can lead to distruptions and other unintended behavior.

```

Comment:
Excellent! There's enough detail here to be specific but not so technical as to be inaccessible.